auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/GrpcUmsClient.java

```diff
@@ -511,7 +512,7 @@ public boolean isAuthorizationEntitlementRegistered(String actorCrn, String acco
         return getAccountDetails(actorCrn, accountId, MDCUtils.getRequestId()).getEntitlementsList()
                 .stream()
                 .map(e -> e.getEntitlementName().toUpperCase())
-                .anyMatch(e -> e.equalsIgnoreCase("CB_AUTHZ_POWER_USERS"));
+                .anyMatch(e -> e.equalsIgnoreCase(Entitlement.CB_AUTHZ_POWER_USERS.name()));
```

Comment:
Entitlement check for authz entitlement still not logged here. The main reason for jira task creation was to use EntitlementService here and also resolve circular dependency between GrpcUmsClient and EntitlementService

Reply:
Yes, this is the reason why it is in do_not_merge state, but I'll put it into draft to make it more clear.

Reply:
I'll leave it simply in DO_NOT_MERGE.

Reply:
oh, ok, yes, makes sense to make it draft for now